inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -39,9 +39,27 @@ class MeshResolution(Enum):
 class WindTunnel(Scenario):
     """Physical scenario of a configurable wind tunnel simulation.
 
-    In this scenario, an object is inserted in a wind tunnel described by the
-    user. The object is then subject to an air flow determined for which the
-    direction and magnitude is defined by the user.
+    A wind tunnel is a tool used in aerodynamic research to study the effects
+    of air moving past solid objects. Here, the tunnel consiste of a box
+    object in 3D space, where the air will flow with a certain velocity.
+
+    Object O==O is set inside the wind tunnel and air flows in a direction
+    established by the user (e.g. below from left to right):
```

Comment:
"An arbitrary object is placed within the tunnel, such that air flows around it, as illustrated in the schematic below:"